docs/source/deprecated.rst

```diff
@@ -5,6 +5,48 @@ This page lists the features that are deprecated and steps to
 update your configuration to use the replacements (if any).
 
 
+matrix_client optional dependency (since 2.24)
+----------------------------------------------
+In older versions ``matrix_client`` was used as matrix provider.
+This optional dependency is now replaced with ``matrix-nio`` which is both
+more featureful (support encrypted rooms) and more actively maintained.
+
+Configuration is backwards compatible so after updating to 2.23 you
+only need to install ``matrix-nio`` and everything should work as previously.
+
+To use encryption you need to update configuration in following ways:
+
+#. Remove ``access_token`` field
```

Comment:
Should we warn when loading a config that still has `access_token` set?

Reply:
https://github.com/thp/urlwatch/pull/620/files#diff-5ee1abebe5480db745909e862aaa226d8f66babc4ab506b526c49cf7c0059fe0R1034-R1038